src/js/components/Grommet/doc.js

```diff
@@ -28,6 +28,10 @@ export const doc = Grommet => {
     theme: PropTypes.object.description(
       'Custom styles for Grommet app component.',
     ),
+    themeMode: PropTypes.oneOf(['dark', 'light']).description(
+      `Dark vs. light theme variation. Default is unspecified and left to
+      theme.`,
+    ),
```

Comment:
Few thoughts here.
Is there a way to set the default `themeMode` from the theme?
How will this behave if a theme isn't supporting `dark` or `light`?

Reply:
Good questions. If a theme doesn't support dark/light, `themeMode` has no effect. I've updated how the default `theme.dark` is calculated to be clearer as well.